hubblestack/daemon.py

```diff
@@ -50,8 +50,11 @@ def run():
         sys.exit(0)
 
     if __opts__['daemonize']:
+        check_pidfile(kill_other=True)
         salt.utils.daemonize()
         create_pidfile()
+    else:
```

Comment:
Can you please add a check to see if it's a single function run or not?

https://github.com/hubblestack/hubble/blob/develop/hubblestack/daemon.py#L81-L84

If you're just running a single function, rather than running the scheduler forever, I want to execute the single function even if hubble is already running on the system.

Reply:
I put the pidfile check next to the pidfile drop... it's only applicable when invoked in daemon mode. I can double check to see if it works that way, but I'm pretty sure.

Reply:
oh, nm, I see. I set it to refuse to run if there's a daemon running without checking function.

Reply:
I'm not convinced it really needed the single function check, but I added it just the same. Confirmed that if hubble is already running and I run the thing with `-d`, it replaces the running process with the new one. Also, if I run it `hubble grains.get fqdn` it just returns the grain and leaves the running process alone.